tests/ui/test_pvc_ui.py

```diff
@@ -121,20 +137,84 @@ def test_create_delete_pvc(
             f"\n actual volume mode:{pvc[0].get_pvc_vol_mode}"
         )
 
+        # Verifying PVC via UI
         logger.info("Verifying PVC Details via UI")
         pvc_ui_obj.verify_pvc_ui(
             pvc_size=pvc_size,
             access_mode=access_mode,
             vol_mode=vol_mode,
             sc_type=sc_type,
+            pvc_name=pvc_name,
+            project_name=project_name,
         )
         logger.info("PVC Details Verified via UI..!!")
 
+        # Creating Pod via CLI
+        logger.info("Creating Pod")
+        if sc_type in (
+            constants.DEFAULT_STORAGECLASS_RBD_THICK,
+            constants.DEFAULT_STORAGECLASS_RBD,
+        ):
+            interface_type = constants.CEPHBLOCKPOOL
+        else:
+            interface_type = constants.CEPHFILESYSTEM
+        new_pod = helpers.create_pod(
+            interface_type=interface_type,
+            pvc_name=pvc_name,
+            namespace=project_name,
+            raw_block_pv=vol_mode == constants.VOLUME_MODE_BLOCK,
+        )
+
+        logger.info(f"Waiting for Pod: state= {constants.STATUS_RUNNING}")
+        wait_for_resource_state(resource=new_pod, state=constants.STATUS_RUNNING)
+
+        # Deleting Pod
+        teardown_factory(new_pod)
+
+        # Expanding the PVC
+        logger.info("Pvc Resizing")
+        new_size = int(pvc_size) + 1
+        pvc_ui_obj.pvc_resize_ui(
+            pvc_name=pvc_name, new_size=new_size, project_name=project_name
+        )
+
+        assert new_size > int(
+            pvc_size
+        ), f"New size of the PVC cannot be less than existing size: new size is {new_size})"
+
+        ocp_version = get_ocp_version()
+        self.pvc_loc = locators[ocp_version]["pvc"]
+
+        # Verifying PVC expansion
+        logger.info("Verifying PVC resize")
+        expected_capacity = f"{new_size} GiB"
+        text_found = pvc_ui_obj.wait_until_expected_text_is_found(
+            format_locator(self.pvc_loc["new-capacity"], expected_capacity),
+            expected_text=expected_capacity,
+            timeout=300,
+        )
+
+        pvc_ui_obj.verify_pvc_resize_ui(expected_capacity=expected_capacity)
```

Comment:
Please add a return to the function verify_pvc_resize_ui True or False if it is verified or not and set here a variable to catch the return result and deal with it.

Reply:
please check